Source/UI/Validator/PhoneNumberValidator.swift

```diff
@@ -1,12 +1,13 @@
 import Foundation
 
 class PhoneNumberValidator: Validator {
-    func validate(text: String?) -> Bool {
-        isEmpty(text: text)
-    }
+    private let maxDigitLength = 25
+    private let minDigitLength = 6
 
-    private func isEmpty(text: String?) -> Bool {
-        text?.isEmpty ?? false
+    func validate(text: String?) -> Bool {
+        guard let text = text else { return true }
```

Comment:
if text is nil we should return false ?, then it would mean if validate returns true for a good data and validate returns false for wrong data. @alex-ioja-yang-cko @ehab-al-cko suggestions ?

Reply:
Fixed in new commit. naming was confusing.